apps/src/gamelab/AnimationTab/AnimationSequenceList.jsx

```diff
@@ -18,7 +18,7 @@ var AnimationSequenceList = React.createClass({
 
   render: function () {
     return (
-      <ScrollableList className="animation-sequence-list">
+      <ScrollableList style={{flex: '1 0 0'}} className="animation-sequence-list">
```

Comment:
Styleguide calls for not inlining these, though maybe we should say it's okay for simple one prop styles? I don't feel too strongly either way.